steve/scrapers.py

```diff
@@ -27,7 +25,7 @@ class YoutubeScraper(object):
     def transform_item(self, item):
         """Converts youtube-dl output to richard fields"""
         return {
-            'title': item['fulltitle'],
+            'title': item['title'],
```

Comment:
Looks like `"fulltitle"` and `"title"` differ when title is truncated: https://github.com/rg3/youtube-dl/blob/b3613d36da14ab527166326707c0f911d192144d/youtube_dl/YoutubeDL.py#L1400

but I haven't found `"fulltitle"` when using a `ydl`